File: api-openbis-javascript/src/v3/imaging/dto/ImagingExportIncludeOptions.js
define([ "stjs", "as/dto/common/Enum" ], function(stjs, Enum) {
	var ImagingExportIncludeOptions = function() {
		Enum.call(this, [ "IMAGE", "RAW_DATA" ]);
	};
	stjs.extend(ImagingExportIncludeOptions, Enum, [ Enum ], function(constructor, prototype) {
	}, {});
	return new ImagingExportIncludeOptions();
})

File: api-openbis-java/source/java/ch/ethz/sis/openbis/generic/imagingapi/v3/dto/ImagingExportIncludeOptions.java
package ch.ethz.sis.openbis.generic.imagingapi.v3.dto;

import ch.systemsx.cisd.base.annotation.JsonObject;

@JsonObject("imaging.dto.ImagingExportIncludeOptions")
public enum ImagingExportIncludeOptions
{
    IMAGE, RAW_DATA;
}

